eng/Packages.Data.props

```diff
@@ -97,6 +97,7 @@
     <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.2.0" />
     <PackageReference Update="Azure.Security.KeyVault.Keys" Version="4.2.0" />
     <PackageReference Update="Azure.Security.KeyVault.Certificates" Version="4.2.0" />
+	<PackageReference Update="Azure.Storage.Common" Version="12.15.0" />
```

Comment:
@weshaggard @tg-msft, should the other storage packages also be updated to use a package reference? Currently I believe this is always pulled in as a project reference.

Reply:
@schaabs, the base storage packages (including Common) always ship together.  My understanding is that the release pipeline automatically converts project references to packages references when we release.

The reason we are running into this issue is that we'd like the ability to ship DM Lib separately from the base storage packages.